sdcm/sct_events.py

```diff
@@ -184,6 +184,17 @@ def __str__(self):
         return "{0}: message={1.message}".format(super(InfoEvent, self).__str__(), self)
 
 
+class CriticalEvent(SctEvent):
```

Comment:
It should be `SchemaValidatorEvent` and if error happens it's severity should be critical.
Look on `DisruptionEvent` as example

Reply:
fixed